Why is "@babel/preset-env" not included in package.json?

I'm getting an error running `ember serve` after installing `ember-auto-import`:
```
Cannot find module '@babel/preset-env'
```
Upon further inspection, it seems this file is responsible for loading that babel module (https://github.com/ef4/ember-auto-import/blob/master/packages/ember-auto-import/ts/webpack.ts#L146). The problem is, this module is not part of the package.json and so I'm missing the dependency.

Should I make a PR? Am I missing something here?

I'm on `ember-auto-import` version `1.5.1`

--- a/packages/ember-auto-import/package.json
+++ b/packages/ember-auto-import/package.json
@@ -27,6 +27,7 @@
   },
   "dependencies": {
     "@babel/core": "^7.1.6",
+    "@babel/preset-env": "^7.0.0",
     "@babel/traverse": "^7.1.6",
     "@babel/types": "^7.1.6",
     "@embroider/core": "^0.4.3",

